eng/source-build-patches/0034-Exclude-Microsoft.Extensions.Hosting.WindowsServices.patch

```diff
@@ -0,0 +1,25 @@
+From 6521f8a43f5c146efc72b72dd3b0e61344a71618 Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Mon, 15 Mar 2021 19:00:55 +0000
+Subject: [PATCH] Exclude Microsoft.Extensions.Hosting.WindowsServices from
+ source-build
+
+---
+ .../Directory.Build.props                                       | 3 ++
+ 1 file changed, 3 insertions(+)
+
+diff --git a/src/libraries/Microsoft.Extensions.Hosting.WindowsServices/Directory.Build.props b/src/libraries/Microsoft.Extensions.Hosting.WindowsServices/Directory.Build.props
+index c5f4d2e6da1..5493f19bc19 100644
+--- a/src/libraries/Microsoft.Extensions.Hosting.WindowsServices/Directory.Build.props
++++ b/src/libraries/Microsoft.Extensions.Hosting.WindowsServices/Directory.Build.props
+@@ -2,5 +2,8 @@
+   <Import Project="..\Directory.Build.props" />
+   <PropertyGroup>
+     <PackageTags>hosting</PackageTags>
++    <!-- Source build does not support Windows. -->
++    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
++    <IsPackable>false</IsPackable>
```

Comment:
@dseefeld why is this necessary? Unfortunately the comment or the commit message doesn't provide any details.

Reply:
I think that since the project was only supported on Windows it wasn't necessary for source-build.

Reply:
Does it hurt anything to build it?  We have plenty of other things that are the same and they don't get excluded from source build. https://github.com/dotnet/runtime/tree/main/src/libraries/System.ServiceProcess.ServiceController for example.

Reply:
More than likely it was introducing prebuilts.  Until we get the prebuilt report cleaned up during this ArPow effort it is difficult to detect.  @dseefeld, are you alright with removing the patch until we can prove it is necessary?